src/stk/image/volume.h

```diff
@@ -227,19 +234,24 @@ namespace stk
         // Fills the volume with the specified value
         void fill(const T& value);
 
-        // Returns value at 
+        // Returns value at
         T at(int x, int y, int z, BorderMode border_mode) const;
         T at(int3 p, BorderMode border_mode) const;
 
-        T linear_at(float x, float y, float z, BorderMode border_mode) const;
-        T linear_at(float3 p, BorderMode border_mode) const;
+        // Indices in the voxel grid
+        T linear_at_index(float x, float y, float z, BorderMode border_mode) const;
+        T linear_at_index(float3 p, BorderMode border_mode) const;
+
+        // Point coordinates in image space
+        T linear_at_point(float x, float y, float z, BorderMode border_mode) const;
+        T linear_at_point(float3 p, BorderMode border_mode) const;
 
         VolumeHelper& operator=(const VolumeHelper& other);
         VolumeHelper& operator=(const Volume& other);
 
         const T& operator()(int x, int y, int z) const;
         T& operator()(int x, int y, int z);
-        
+
```

Comment:
This stray whitespace is annoying, I was thinking to do a single PR just to clean it up.

Reply:
Haha, yeah, or just clean them up as you go. Doesn't really affect the output so go ahead.

Reply:
I have it highlighted by the editor, since in most projects stray whitespace is deprecated, so it's always punching me in the eyes. But it's also bad practice to clutter the diff with unrelated whitespace fixes, so maybe I will take care of it all at once...